core/settings/qPrefDiveComputer.h

```diff
@@ -35,6 +35,8 @@ class qPrefDiveComputer : public QObject {
 	Q_PROPERTY(QString vendor3 READ vendor3 WRITE set_vendor3 NOTIFY vendor3Changed)
 	Q_PROPERTY(QString vendor4 READ vendor4 WRITE set_vendor4 NOTIFY vendor4Changed)
 
+	Q_PROPERTY(bool sync_dive_computer_time READ sync_dive_computer_time WRITE set_sync_dive_computer_time NOTIFY sync_dive_computer_timeChanged)
```

Comment:
maybe `sync_dc_time` or something? That's a bit longish...

Reply:
I used the pattern that exists in `prefs.h`, but no problem with making it shorter:
https://github.com/subsurface/subsurface/pull/3857/files#diff-18ce2a282c1f6acc704513f0cc9efa1b7d10cc7dd186305c6ea7368c433992c7L98